tests/trainer/test_fsdp_checkpoint.py

```diff
@@ -743,13 +746,19 @@ def test_fsdp_partitioned_state_dict_load(
     weights_only: bool,
     load_ignore_keys: Union[list[str], None],
     use_symlink: bool,
+    use_tp: bool,
     use_remote,
     s3_bucket,
     s3_ephemeral_prefix,
     request,
 ):
+    if use_tp:
+        pytest.skip('TP does not work with DP dimension 1')
```

Comment:
the TP test is always skipped ?  We can only run 1-GPU test at this moment, is my understanding correct ?

Reply:
Yea, we need 4 GPU tests to support. @KuuCi  is adding soon!